src/js/components/FragmentList/fragmentList.style.jsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+const FragmentList = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+`;
+
+const SortingWrapper = styled.div`
+    display: flex;
+`;
+
+const SortingButton = styled.button`
+      border: 0;
+      margin: 2px;
+      padding: 2px;
+      color: ${({ theme }) => theme.textColor};
+      background-color: ${({ theme }) => theme.buttonBgColor};
+      &:active {
+        background-color: ${({ theme }) => theme.fragmentHighlight};
+      }
+`;
+
+const SortingButtonStatus = styled(SortingButton)`
+      width: 15px;
+`;
+
+const SortingButtonId = styled(SortingButton)`
```

Comment:
Aren't these two buttons the same? Perhaps this can be renamed to something more generic